dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -583,8 +583,8 @@ ANNO_TARGETS_CACHE_EXCEPTION.useraction=For more information, see the exception
 # # {0} A string: A class name.
 # # {1} A string: A location which contains the class: The path to the JAR file which contains the class, or the path to the directory which contains the class, or the name of the internal data structure which contains the class.
 # # {2} A string: Message text from an exception.
-ANNO_TARGETS_CLASS_SCAN_EXCEPTION: CWWKC0103W: An exception occurred while scanning class {0} of {1}.  The exception was {2}.
-ANNO_TARGETS_CLASS_SCAN_EXCEPTION.explanation: The class and annotations data could not be scanned. The associated module may function incorrectly.
+ANNO_TARGETS_CLASS_SCAN_EXCEPTION=CWWKC0103W: An exception occurred while scanning class {0} of {1}.  The exception was {2}.
+ANNO_TARGETS_CLASS_SCAN_EXCEPTION.explanation=The class and annotations data could not be scanned. The associated module may function incorrectly.
```

Comment:
Change
"The class and annotations data could not be scanned. The associated module may function incorrectly."
to
"The class and annotations data was not able to be scanned. The associated module might function incorrectly."